book/src/getting-started/1-qobjects-in-rust.md

```diff
@@ -27,32 +27,31 @@ Therefore it is an ideal candidate to replace C++ for writing the back-end busin
 
 However, C++ as well as QML still have their place in Qt applications.
 For that reason Rust, C++, and QML should all be able to be used to complement each other.
-CXX-Qt aims to make it easy to integrate all three languages easily with each other, through the use of Qt's [meta object system](https://doc.qt.io/qt-5/metaobjects.html).
-
-Qt's design is inherently object-oriented, which is true both for C++ and QML as well.
-Therefore, in order to integrate well with Qt, Rust needs to be able to extend the Qt object system with its own QObject subclasses and instances.
-This is exactly what CXX-Qt allows you to do.
-
-As Rust doesn't offer classes with inheritance and polymorphism, CXX-Qt uses macros when defining new QObject subclasses.
-
-These CXX-Qt modules consist of multiple parts:
-- A struct marked with a `#[cxx_qt::qobject]` macro
-    - Defines the C++ QObject subclass name
-    - A normal Rust struct.
-    - One struct instance is created per class instance.
-    - Defines which Properties will be in the QObject subclass.
-    - Can contains any Rust-only data.
-    - Needs to implement the `Default` trait.
-- The `impl` of the `#[cxx_qt::qobject]` marked struct (optional):
-    - Contains any Rust code.
-    - Functions marked with `#[qinvokable]` will be callable from QML and C++.
-- The `Signal` enum
-    - A normal Rust enum.
-    - Defines signals that are added to the QObject class
+CXX-Qt aims to make it easy to integrate all three languages with each other, through the use of Qt's [meta object system](https://doc.qt.io/qt-5/metaobjects.html) and [CXX](https://cxx.rs).
+
+As CXX-Qt is largely an expansion on top of CXX, you should make yourself familiar with CXX first.
+You can read their excellent documentation here:
+> [https://cxx.rs/](https://cxx.rs)
+
+CXX already provides idiomatic interaction between C++ and Rust, from Rust, as well as from C++.
+CXX-Qt builds on this foundation, to allow you to define Qt-specific concepts in Rust.
+
+These concepts include:
+- Custom QObject classes
+- Properties
+- Signals
```

Comment:
should signals be last, then it's the same order as the list below?

Reply:
Yup, added the indentation as well.